collector/Version.h.in

```diff
@@ -1,10 +1,10 @@
-#ifndef _VERSION_H_
-#define _VERSION_H_
+#ifndef _COLLECTOR_VERSION_H_
+#define _COLLECTOR_VERSION_H_
 
-#define COLLECTOR_VERSION "${COLLECTOR_VERSION}"
+#define COLLECTOR_VERSION "@COLLECTOR_VERSION@"
```

Comment:
Is there any difference between `@@` and `${}`? I actually like `${}` a bit more, but it's just a personal preference.

Reply:
Evidently there is no difference. I felt like @@ was more cmake native, but see no indication in the docs. Leaving it as is.